src/utils/contracts/sputnik-dao.ts

```diff
@@ -0,0 +1,167 @@
+import { tx, view } from "../wallet";
+
+
+const FACTORY_ADDRESS_SELECTOR: Record<string, string> = {
```

Comment:
You know about objects right? 😆

Reply:
Used records to specify a type on keys+values